src/MOCKernel.cpp

```diff
@@ -0,0 +1,165 @@
+#include "MOCKernel.h"
+#include "TrackGenerator.h"
+
+/**
+ * @brief Constructor for the MOCKernel assigns default values
+ * @param track_generator the TrackGenerator used to pull relevant tracking
+ *        data from
+ * @param row_num the row index into the temporary segments matrix
+ */
+MOCKernel::MOCKernel(TrackGenerator* track_generator, int row_num) {
+  _count = 0;
+  _max_tau = track_generator->retrieveMaxOpticalLength();
+}
+
+
+/**
+ * @brief Constructor for the VolumeKernel assigns default values, calls
+ *        the MOCKernel constructor, and pulls refernces to FSR locks and FSR
+ *        volumes from the provided TrackGenerator.
+ * @param track_generator the TrackGenerator used to pull relevant tracking
+ *        data from
```

Comment:
Ok, I see that it isn't an incomplete comment, but it could be reworded to make it easier to read perhaps.